BT.playground/Contents.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+//cau 1 Viết hàm để check số đã nhập có phải là số có các chữ số tăng dần hay không
+func cau1(n:Int)->Void{
+    let arr:String = String(n)
+    let result = Int(String(arr.sorted()))
+    if result == n{
```

Comment:
Để ý dấu cách cho đúng, viết liền nhau gây khó đọc và chật chội
func mà -> Void thì ko cần phải viết ra
```suggestion
func cau1(n: Int) {
```